noetic/distribution.yaml

```diff
@@ -155,6 +155,21 @@ repositories:
       url: https://github.com/astuff/astuff_sensor_msgs.git
       version: master
     status: maintained
+  async_web_server_cpp:
+    doc:
+      type: git
+      url: https://github.com/fkie/async_web_server_cpp.git
+      version: master
```

Comment:
This branch doesn't match the source branch.  They don't *have* to match, but it is much more common for them to match.  Did you mean this?

Reply:
I use the `develop` branch for development and merge the releases into `master`. I assumed that it is more convenient/useful to have the release versions in the documentation branch.

Reply:
OK, it is no problem to have them be different, I just wanted to check that you intended that.

Reply:
I will probably rename the `master` branch to `releases` at some point, to mirror the structure I just set up for ROS 2. But for now, this will be fine.